qiskit/optimization/algorithms/admm_optimizer.py

```diff
@@ -76,6 +77,8 @@ def __init__(self,
             tau_decr: Parameter used in the rho update (UPDATE_RHO_BY_RESIDUALS).
             mu_res: Parameter used in the rho update (UPDATE_RHO_BY_RESIDUALS).
             mu_merit: Penalization for constraint residual. Used to compute the merit values.
+            warm_start: Start ADMM with pre-initialized values for binary and continuous variables
```

Comment:
Do you think some form of guidance around warm start and when it might be used etc. might be useful in the main docstring of the class here. I see it's set False by default so I guess, in terms of the result, its something a user needs to be more selective around turning it on?

Reply:
Ok, I'll try to express when it can be useful. It's sort of a last resort if other options don't work and we did not heavily tested it.

Reply:
Slightly updated the documentation on this option.